app/lib/frontend/handlers.dart

```diff
@@ -169,11 +175,15 @@ Future<shelf.Response> searchHandler(shelf.Request request) async {
 
   final resultPage = await searchService.search(query, useService);
   final links = new SearchLinks(query, resultPage.totalCount);
+  final String content = templateService.renderSearchPage(resultPage, links);
+
   _searchBackendTracker.add(resultPage.backend);
   _searchBackendLatencyTracker.add(resultPage.latency);
   _searchOverallLatencyTracker.add(sw.elapsed);
 
-  return htmlResponse(templateService.renderSearchPage(resultPage, links));
+  // do not await on memcache update
+  searchMemcache?.setUiSearchPage(cacheUrl, content);
```

Comment:
I think it's better to await for the memcache update for two reasons: Firstly the additional latency is negligible in comparison to the rest of what the search takes (maybe around 1-2 ms), secondly if memcache isn't working or throws an error we will only see it in the logs if the request wasn't done. (If the request handler is done, the error won't show up on the request log entry.)